<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>  Левый нижний угол:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>   Правый верхний угол:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x1Input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="y1Input">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="x2Input">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="y2Input">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setRectangleButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать лист</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>801</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вырезать клетку:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xInput">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yInput">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cutCellButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Вырезать</string>
    </property>
   </widget>
   <widget class="QLabel" name="piecesCountLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Кусков: 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Восстановить клетку:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x0Input">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="y0Input">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>70</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="restoreCellButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Восстановить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
